samples/nats_jetstream/v3/sender/main.go

```diff
@@ -101,3 +103,11 @@ func createStream(url, streamName, subjectName string) error {
 	_, err = js.CreateOrUpdateStream(ctx, streamConfig)
 	return err
 }
+
+func buildSubject(base string, i int) string {
```

Comment:
nit: suggesting

```suggestion
func getRandomSubject(base string, i int) string {
```

Reply:
The subject is not random so that would not be a great name.

Reply:
Fair